packages/interface/src/components/complementary-area/index.js

```diff
@@ -144,6 +148,7 @@ function ComplementaryArea( {
 								isActive && ( ! showIconLabels || isLarge )
 							}
 							aria-expanded={ isActive }
+							disabled={ isActive === undefined }
```

Comment:
I believe it's recommended to use `aria-disabled` for a11y so that disabled buttons can receive focus.

Reply:
https://github.com/WordPress/gutenberg/pull/47498/commits/0af516e80885a61b369501cb1e6029706099ce61